lms/djangoapps/learner_home/test_utils.py

```diff
@@ -86,3 +88,19 @@ def create_test_enrollment(user, course_mode=CourseMode.AUDIT):
     test_enrollment.course_overview.end = random_date()
 
     return test_enrollment
+
+
+def expected_progress_url(course_id, course_home_mfe_progress_tab_is_active):
+    """
+    Returns expected course progress URL based on the status of the course_home_mfe_progress_tab_is_active.
+
+    :param course_id: The course key / ID for which the progress url is being requested.
+    :param course_home_mfe_progress_tab_is_active: A boolean matching the return value from the
+                                                   course_home_mfe_progress_tab_is_active function,
+                                                   implemented in lms.djangoapps.course_home_api.toggles.
+
+    :return: The course progress URL.
+    """
+    if course_home_mfe_progress_tab_is_active:
+        return f'{settings.LEARNING_MICROFRONTEND_URL}/course/{course_id}/progress'
+    return f'/courses/{course_id}/progress'
```

Comment:
```suggestion
    return reverse("progress", kwargs={"course_id": course_id})
```

You should use the reverse function here if possible.